web-page-gen.rkt

```diff
@@ -177,15 +312,15 @@
 (define create-heading-test "Create heading level 2 \"Heading text \" color blue, font Times New Roman, size 14, align right")
 
 ;; Helper tests
-(test (extract-text-from-create-statement create-paragraph-test) "Lorem ipsum. ")
-(test (extract-text-from-create-statement create-heading-test) "Heading text ")
-(test (extract-heading-level-from-create-statement create-heading-test) "2")
-(test (extract-style create-paragraph-test) "color red, font Comic Sans MS, size 12, align center")
-(test (extract-style-item "color red") "red")
-(test (extract-style-item "font comic sans") "comic sans")
-(test (extract-alignment "align center") (align-center))
-(test (extract-alignment "align right") (align-right))
-(test (extract-alignment "align left") (align-left))
+;(test (extract-text-from-create-statement create-paragraph-test) "Lorem ipsum. ")
+;(test (extract-text-from-create-statement create-heading-test) "Heading text ")
+;(test (extract-heading-level-from-create-statement create-heading-test) "2")
+;(test (extract-style create-paragraph-test) "color red, font Comic Sans MS, size 12, align center")
+;(test (extract-style-item "color red") "red")
+;(test (extract-style-item "font comic sans") "comic sans")
+;(test (extract-alignment "align center") (align-center))
+;(test (extract-alignment "align right") (align-right))
+;(test (extract-alignment "align left") (align-left))
 
 ;; Parse tests
 ;; Parse one expression
```

Comment:
Im assuming the parse and interp tests still pass?

Reply:
Yes, the original parse function is commented out, I replaced it with using the combinators. I just commented out the helper tests since the parser no longer uses them.